docs/internal.yml

```diff
@@ -0,0 +1,245 @@
+openapi: 3.0.3
+
+info:
+  title: Azure Iot Manager
+  description: |
+    Internal API for managing devices on Azure IoT Hub.
+
+  version: "1"
+
+servers:
+  - url: http://mender-azure-iot-manager:8080/api/internal/v1/azure-iot-manager
+
+tags:
+  - name: Internal API
+
+paths:
+  /health:
+    get:
+      tags:
+        - Internal API
+      summary: Get health status of service
+      operationId: Check Health
+      responses:
+        204:
+          description: Service is healthy.
+        400:
+          description: Invalid Request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        500:
+          description: Internal Server Error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /alive:
+    get:
+      tags:
+        - Internal API
+      summary: Get service liveliness status.
+      operationId: Check Liveliness
+      responses:
+        204:
+          description: Service is up and serving requests.
+        500:
+          description: Internal Server Error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /tenants/{tenantId}/devices:
+    post:
+      tags:
+        - Internal API
+      operationId: Provision device
+      summary: Register a new device with the deviceconfig service.
+      parameters:
+        - in: path
+          name: tenantId
+          schema:
+            type: string
+          required: true
+          description: ID of tenant the device belongs to.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/NewDevice'
+      responses:
+        204:
+          description: Device provisioned successfully.
+        400:
+          $ref: '#/components/responses/InvalidRequestError'
```

Comment:
409 Conflict undocumented

Reply:
Thanks for the sharp observation, I'll update the docs.